t/17-build_tagging.t

```diff
@@ -63,6 +63,7 @@ my $job_hash = {
     group_id => 1001
 };
 
+
```

Comment:
```suggestion
```

I think generally one new line between subtests is enough - and that's also the usual amount of spacing we use in this repo.